rag/export_training_data.py

```
@@ -1,185 +1,186 @@
+
+    # Emails
+    text = re.sub(r"[\w\.-]+@[\w\.-]+\.[A-Za-z]{2,}", "<EMAIL>", text)
+
+    # URLs
+    text = re.sub(r"https?://\S+", "<URL>", text)
+
+    # Phone numbers (simple heuristic: international +81 or sequences of digits with separators)
+    text = re.sub(r"\+?\d[\d\-\s()]{6,}\d", "<PHONE>", text)
+
+    # 名前フィールド: 氏名: 山田太郎 -> 氏名: <PERSON>
+    text = re.sub(r"((?:氏名|名前|Name|Full ?Name)[:：])\s*[^\n,。;]+", r"\1 <PERSON>", text)
```

Ruff (0.16.1)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)